BotRoyaleUnity/Assets/Scripts/BuildRobot.cs

```diff
@@ -84,6 +85,8 @@ GameObject build(string jsonString)
             index++;
         }
         parent.AddComponent<RoombaMovement>();
+        Rigidbody parentRigidbody = parent.AddComponent<Rigidbody>();
+        parentRigidbody.mass = 5;
```

Comment:
since there's a bunch of configuration when instantiating a robot here, it could be worth it to make it into a prefab with the RoombaMovement and RIgidbody components already configured.